app-frontend/src/app/pages/settings/settings.html

```diff
@@ -0,0 +1,15 @@
+<div class="container column-stretch dashboard">
+  <div class="sidebar">
+    <div class="sidebar-scrollable">
+      <nav>
+        <a ui-sref="settings.profile" ui-sref-active="active">Profile</a>
+        <a ui-sref="settings.account" ui-sref-active="active">Account</a>
+        <a ui-sref="settings.tokens" ui-sref-active="active">API tokens</a>
```

Comment:
I left a couple prototypes out of this, such as the detail view for the buckets in the library view since I didn't think that it would be very helpful to include them until we have a good idea of how the information will look and at least implement the list view.  It's pretty easy to add them though, if you just follow the pattern that I did for the rest of the views.